app/javascript/src/components/Projects/List/Header.tsx

```diff
@@ -1,69 +1,85 @@
 import * as React from "react";
+import projectApi from "apis/projects";
+import AutoComplete from "common/AutoComplete";
+import Logger from "js-logger";
 import { Funnel, MagnifyingGlass, Plus } from "phosphor-react";
 
-const Header = ({ setShowProjectModal, isAdminUser }) => (
-  <div
-    className={
-      isAdminUser
-        ? "sm:flex mt-6 mb-3 sm:items-center sm:justify-between"
-        : "sm:flex mt-6 mb-3 sm:items-center"
+const Header = ({ setShowProjectModal, isAdminUser }) => {
+
+  const searchCallBack = async (searchString, setDropdownItems) => {
+    try {
+      const res = await projectApi.get();
+      const dropdownList = res.data.projects.map((project) => ({
+        label: project.name,
+        value: project.id
+      }));
+      const searchList = dropdownList.filter(item => item.label.toLowerCase().includes(searchString.toLowerCase()));
+      setDropdownItems(searchList);
+    } catch (err) {
+      Logger.error(err);
     }
-  >
-    <h2 className="header__title">Projects</h2>
-    <React.Fragment>
-      <div className="header__searchWrap">
-        <div className="header__searchInnerWrapper">
-          <input
-            type="search"
-            className="header__searchInput"
-            placeholder="Search"
-          />
-          <button className="absolute inset-y-0 right-0 pr-3 flex items-center cursor-pointer">
-            <MagnifyingGlass size={12} />
-          </button>
+  };
+
+  return (
+    <div
+      className={
+        isAdminUser
+          ? "sm:flex mt-6 mb-3 sm:items-center sm:justify-between"
+          : "sm:flex mt-6 mb-3 sm:items-center"
+      }
```

Comment:
```suggestion
      className={`sm:flex mt-6 mb-3 sm:items-center  ${isAdminUser ? "sm:justify-between" : " "}` }
      }
```

avoid repeating the classes while using ternary operator.